user/handler.go

```diff
@@ -54,24 +54,45 @@ func Register(w http.ResponseWriter, r *http.Request) {
 
 }
 
+type BookTimeData struct {
+	Id   string `json:"id"`
+	Time string `json:"time"`
+}
+
 func BookTime(w http.ResponseWriter, r *http.Request) {
```

Comment:
we need to store the user id of the person who booked the time slot.
e.g. Alice booked time in Bob's calendar.